### PR TITLE
Abductor walls can now smooth with most windows

### PR DESCRIPTION
**Pull request by 1080pCat**
## What Does This PR Do

Allows the abductor walls to smooth to most windows

## Why It's Good For The Game

Before this using abductor walls for anything but making enclosed rooms was rather difficult because having windows made it look really ugly, this changes that so it looks a bit better.

## Images of changes

# Before 
B6NhU0Mhgj

# After
dreamseeker_Z36y3SECfh

## Testing

see images above

<hr>

### Declaration
- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/feature_freeze/CODE_OF_CONDUCT.md#currently-changes-to-the-following-types-of-content-reuires-pre-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.

<hr>

## Changelog
:cl:
tweak: Abductor walls can now smooth with most non standard windows
/:cl:

### CODE REVIEW

**Comment by DGamerL** [comment]
Not sure on most of these, wouldn't a better solution be to have most windows not try to smooth with abductor windows at all?

Also Brass windows are not made to be smoothed it seems lmao

**Comment by 1080pCat** [comment]
> Also Brass windows are not made to be smoothed it seems lmao

yeah blame whoever put brass windows in the full tiles section

**Comment by Christasmurf** [comment]
Okay so, to explain the addition then removal of the approval tag:
1. Not the greatest look, but that's hardly in your control as you're only touching the code.
2. Codewise, I thought this would be useful with the upcoming walls resprite. However, within that resprite, every single wall combination should be able to smooth with very minor exceptions to shuttle wall types, which will only be able to smooth with themselves to avoid clipping issues when docking with the station. Due to this, the abductor wall does not need to be able to smooth with normal window types, as well as plastitanium glass not needing to smooth with anything but the matching wall. You changing the code now will likely end up being reverted as the spriting changes are made.
3. After dev discussion it would probably be in our best interest to add abductor glass minerals and walls to serve the purpose you're trying to achieve here while keeping the shuttle walls (any wall that smooths diagonally) contained to only smoothing with itself and it's associated window type. If you would instead desire to work the functionality for those, I would be happy to assist with the sprites.

TLDR: While I would like to allow smoothing between these wall types, it'd likely cause too much difficulty for shuttles/sprites in the long run, and these walls were clearly NOT sprited with the intention of smoothing with other walls/windows as the shape does not work great.

**Comment by 1080pCat** [comment]
> Okay so, to explain the addition then removal of the approval tag:
> 
>     1. Not the greatest look, but that's hardly in your control as you're only touching the code.
> 
>     2. Codewise, I thought this would be useful with the upcoming walls resprite. However, within that resprite, every single wall combination should be able to smooth with very minor exceptions to shuttle wall types, which will only be able to smooth with themselves to avoid clipping issues when docking with the station. Due to this, the abductor wall does not need to be able to smooth with normal window types, as well as plastitanium glass not needing to smooth with anything but the matching wall. You changing the code now will likely end up being reverted as the spriting changes are made.
> 
>     3. After dev discussion it would probably be in our best interest to add abductor glass minerals and walls to serve the purpose you're trying to achieve here while keeping the shuttle walls (any wall that smooths diagonally) contained to only smoothing with itself and it's associated window type. If you would instead desire to work the functionality for those, I would be happy to assist with the sprites.
> 
> 
> TLDR: While I would like to allow smoothing between these wall types, it'd likely cause too much difficulty for shuttles/sprites in the long run, and these current walls were clearly NOT sprited with the intention of smoothing with other walls/windows as the shape does not work great.

closing due to the valid points made here.
hopefully the walls resprite comes around with better alien walls soon, as they're not too useful in building as they're jut out weirdly unsmoothed
